hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -1472,7 +1470,7 @@ private void waitOnAllRegionsToClose(final boolean abort) {
     }
   }
 
-  private static boolean sleep(long millis) {
+  private static boolean sleeps(long millis) {
```

Comment:
`sleep` is good enough right? why `sleeps`?

Reply:
sleep is clash with Thread.

Reply:
Oh yes, missed it. Let's name it `sleepInterrupted` because it makes thread sleep and if interrupted, returns true, else false.

Reply:
Good idea, thanks.